advanced/Scripts/update.sh

```diff
@@ -91,12 +91,37 @@ GitCheckUpdateAvail() {
 
   # Fetch latest changes in this repo
   git fetch --quiet origin
```

Comment:
See [line #140](https://github.com/pi-hole/pi-hole/pull/1096/files#diff-2670cb4d297fe9db4f216b0d88761e45R140)

`if_repo` has to be successful before even going into this routine.